chains/inevm-sepolia-testnet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "inevm-sepolia-testnet",
```

Comment:
Can you provide proof that this chain utilizes Ethereum Sepolia as data availability layer? I couldn't find any.

Reply:
Looks like they are leveraging `Celestia` as DA layer. https://blog.injective.com/en/injective-launches-inevm-on-mainnet-the-first-rollup-to-hyperscale-concurrent-vm-development/ 

Thanks for checking.

Reply:
As a note, we don't care about DA specifically, what matters is where the proofs are posted/where the funds are bridged from